pkg/ccl/backupccl/backup_test.go

```diff
@@ -8268,17 +8266,21 @@ table_name FROM [SHOW TABLES] ORDER BY schema_name, table_name`
 
 	// Check that show tables in one of the restored DBs returns an empty result.
 	sqlDBRestore.Exec(t, "USE d1")
-	sqlDBRestore.CheckQueryResults(t, "SHOW TABLES", [][]string{})
+	sqlDBRestore.CheckQueryResults(t, "SELECT table_name, type, owner FROM [SHOW TABLES]", [][]string{
+		{"t", "table", "root"},
+	})
 
 	sqlDBRestore.Exec(t, "USE d2")
-	sqlDBRestore.CheckQueryResults(t, "SHOW TABLES", [][]string{})
+	sqlDBRestore.CheckQueryResults(t, "SELECT table_name, type, owner FROM [SHOW TABLES]", [][]string{
+		{"t", "table", "root"},
+	})
 
 	testutils.SucceedsSoon(t, func() error {
 		ch <- timeutil.Now()
 		<-finishedCh
 
 		// Check that all the synthesized temp schemas have been wiped.
-		sqlDBRestore.CheckQueryResults(t, checkSchemasQuery, [][]string{})
+		sqlDBRestore.CheckQueryResults(t, checkSchemasQuery, [][]string{{"0"}})
```

Comment:
Won't we have to check in `defaultdb`, `d1` and `d2` now? Currently, I believe this will only check in `d2`.